rasa/shared/core/events.py

```diff
@@ -1475,8 +1475,8 @@ def __init__(
 
     def __repr__(self) -> Text:
         """Returns event as string for debugging."""
-        return "ActionExecuted(action: {}, policy: {}, confidence: {})".format(
-            self.action_name, self.policy, self.confidence
+        return "ActionExecuted(action: {}, policy: {}, confidence: {}, metadata: {})".format(
```

Comment:
This metadata can be quite large. Is it good to always include it in the representation string?